sdk-python/livescoreapi.py

```diff
@@ -7,17 +7,77 @@
 """
 
 import urllib.request
+import requests
 
 
-class livescores:
-    
-    def get_all(key,secret):
-        html_file = urllib.request.urlopen("http://livescore-api.com/api-client/scores/live.json?key=" + key + "&secret=" + secret)
-        json_data = html_file.read()
-        return json_data
+class Livescores:
+
+    api_url = ''
+    api_key = ''
+    api_secret = ''
+
+    def __init__(self, api_url, api_key, api_secret):
+        self.api_url = api_url
+        self.api_key = api_key
+        self.api_secret = api_secret
+
     
+    def get_all_livescores(self):
+       livescores = requests.get('{}scores/live.json?key={}&secret={}'.format(self.api_url, self.api_key, self.api_secret))
+       return livescores.json()
+
+
+    def validate_country(self, country_id):
+        if (isinstance(country_id, str)):
+            raise ValueError("Country ID must be a integer")
+        elif country_id == None:
```

Comment:
This should be a separate `if`with its own exception, you can't raise Errors in some places and then return false in others it should be consistent.

Reply:
`==` None does not exist `is None`